src/components/dateTimePicker/index.tsx

```diff
@@ -41,7 +41,11 @@ export interface DateTimePickerProps {
   /**
    * The onChange callback
    */
-  onChange?: (date: Date) => void;
+  onChange?: (date?: Date) => void;
```

Comment:
This is potentially breaking change...

Reply:
Type change can't be a breaking change, worst thing it can break someone TS check

Reply:
I know, by 'potentially ' I meant that if you return undefined here instead of a Date it can break whoever using it without checking (basically on Android since on iOS we set a new Date if undefined).